nifi-nar-bundles/nifi-tcp-bundle/.gitignore

```diff
@@ -0,0 +1,16 @@
+target
+.project
+.settings
+.classpath
+nbactions.xml
+nb-configuration.xml
+.DS_Store
+.metadata
+.recommenders
+
+# Intellij
+.idea/
+*.iml
+*.iws
+*~
+
```

Comment:
you may want to remove this gitignore since all this stuff is already mentioned in the root one.